Kajabi/sage-lib
# Docs: Icon documentation update (React)

## Description
Updates icon documentation to provide clarity and reduce overall code complexity of examples.


## Screenshots

|  Before  |  After  |
|--------|--------|
|<img width="1687" alt="Screenshot 2023-10-23 at 9 52 06 AM" src="https://github.com/Kajabi/sage-lib/assets/1175111/54bff89a-e91f-45ad-ad9a-fd74f22e3063">|<img width="1689" alt="Screenshot 2023-10-23 at 9 50 48 AM" src="https://github.com/Kajabi/sage-lib/assets/1175111/18e229a8-1cfb-4e50-b3ef-f61800bc7b2b">|


## Testing in `sage-lib`

- Navigate to storybook [icon](http://localhost:4100/?path=/docs/sage-icon--default-story)
- Verify documentation updates.




## Testing in `kajabi-products`

1. (**LOW**) Update to icon react component documentation. No effect on KP expected

## Related
N/A
